pkgs/games/boilr/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, clangStdenv, gtk3, xorg, perl
+, openssl, speechd, libxkbcommon, libGL, wayland }:
+let
+  rpathLibs = [
+    speechd
+    openssl
+    gtk3
+    libxkbcommon
+    libGL
+
+    # WINIT_UNIX_BACKEND=wayland
+    wayland
+
+    # WINIT_UNIX_BACKEND=x11
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    xorg.libX11
+    xorg.libxcb
+  ];
+in rustPlatform.buildRustPackage.override { stdenv = clangStdenv; } rec {
+  pname = "BoilR";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "PhilipK";
+    repo = "BoilR";
+    rev = "v.${version}";
+    sha256 = "sha256-mdkRuEzfWhdbX0DD6uwc1o8kXjqk1Y0+FFKHrM2m81w=";
+  };
+
+  cargoHash = "sha256-5nhtGqukZ8tp7gPV+JiUEHTzWUj6JtsB2i7bODcXJSc=";
+
+  nativeBuildInputs = [ perl ];
+
+  buildInputs = rpathLibs;
+
+  postInstall = ''
+    patchelf --add-rpath "${lib.makeLibraryPath rpathLibs}" $out/bin/boilr
+  '';
+
+  dontPatchELF = true;
+
+  meta = with lib; {
+    description =
+      "BoilR automatically adds (almost) all your games to your Steam library (including image art)";
+    homepage = "https://github.com/PhilipK/BoilR";
+    licenses = with lib.licenses; [ asl20 mit];
+    platforms = platforms.all;
```

Comment:
specify explicitly supported platforms. I doubt this is reall buildable on all (https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix)

Reply:
would it be better to just not specify the platform? according to the manual:

> If no platform is specified, the package does not have prebuilt binaries

Reply:
no, then hydra won't cache it at all